### PR TITLE
Update custom fields docs

### PR DESCRIPTION
**Pull request by tandrup**

### CODE REVIEW

File: api-docs/custom-field-changing-definitions.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	    
+	-   The existing value will be saved for retrieval in the API in the `lastIncompatibleValue` field.
+    
+-   **String**
+    
+	-   If an existing value is longer than the new maximum length, the value will be truncated.
+	    
+	-   The existing value will be saved for retrieval in the API in the `lastIncompatibleValue` field.
+
+	- Regex patterns can be changed, but be warned that existing data can become uncompliant with the new pattern.
```
Comment (csk-trackunit) [inline api-docs/custom-field-changing-definitions.md:45]: What happens to invalid regex values then?

File: api-docs/custom-field-definitions.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+| Field type   | Description                                                                                |
+| ------------ | ------------------------------------------------------------------------------------------ |
+| Boolean      | Boolean value that can store true or false.                                                |
+| Date         | Date field.                                                                                |
+| Dropdown     | A predefined list of options. Supports both single and multi selects.                      |
+| E-mail       | Email field. Will be rendered with a mailto link in the UI.                                |
+| Monetary     | Monetary values with currency in ISO 4217 standard.                                        |
+| Number       | Numeric values. Supports unit conversion between metric and US customary units.            |
+| Phone number | Phone number. Validates the format according to country specific rules, but does not validate the phone number actually exists. |
```
Comment (csk-trackunit) [inline api-docs/custom-field-definitions.md:20]: Consider a `whether` before `the phone number`

File: api-docs/custom-field-definitions.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| Text string  | Free text field. Supports limiting length.                                                 |
+| Web address  | Web address. Will be shown as a link to open a new window in the UI.                       |
+
+It is possible to control if new fields should show up in appropriate places in the user interface or be an API only field.
```
Comment (csk-trackunit) [inline api-docs/custom-field-definitions.md:24]: How?

File: api-docs/custom-field-definitions.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Once a definition has been created values can be set using the [values API](custom-field-values).
+
+# Field definition ownership
+
+Custom field definitions can be owned by one of these:
+
+- **Customer account** - Customer defined fields will be owned by the customer account and require admin permission in that account to change.
```
Comment (csk-trackunit) [inline api-docs/custom-field-definitions.md:32]: I know the title says that this is about the definitions, but perhaps make it explicit anyway. `Customer defined field definitions will be owned...`

File: api-docs/custom-field-definitions.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+When creating a definition using this API it will always be owned by your customer account. It is however possible to retrieve all 3 types using the API.
+
+## Data sharing
+
+Regardless of who owns the definition of a field the values in a field may be shared between accounts.
+
+| Scope type                | Description | 
+|---------------------------|-------------------------------|
+| ACCOUNT                   | Values with account scope will be shared within a single account |
+| ACCOUNT_WRITE_GLOBAL_READ | Updating values will be possible within a single account and visible to all accounts (read) |
```
Comment (csk-trackunit) [inline api-docs/custom-field-definitions.md:45]: I assume that global readability also requires access to the entity?

**Review by csk-trackunit — APPROVED** [review]
Nice 👍 I have only minor comments about grammar and if we should be more explicit some places.

File: api-docs/custom-field-changing-definitions.md (hunk 1 above)
Comment (tandrup) [reply api-docs/custom-field-changing-definitions.md:45]: They are still there

**Pull request merged**